libkrun/libkruntime.c

```diff
@@ -40,4 +41,37 @@ JNIEXPORT jdouble JNICALL
 Java_IterationsRunner_JNI_1clock_1gettime_1monotonic(JNIEnv *e, jclass c) {
     return (jdouble) clock_gettime_monotonic();
 }
+
+JNIEXPORT jlong JNICALL
+Java_IterationsRunner_JNI_1read_1ts_1reg(JNIEnv *e, jclass c)
+{
```

Comment:
I'm using "almost KNF" code style for C. It's KNF with tabs replaced by four spaces. Function braces are on a newline, all others are not.